crates/router/src/core/payments/operations/payment_method_validate.rs

```diff
@@ -307,7 +306,7 @@ impl PaymentMethodValidate {
         storage::PaymentAttemptNew {
             payment_id: payment_id.to_string(),
             merchant_id: merchant_id.to_string(),
-            attempt_id: Uuid::new_v4().simple().to_string(),
+            attempt_id: format!("{payment_id}_1"),
```

Comment:
Can we have a function to generate payment attempt IDs which accepts the payment ID and attempt count?